sync/legacy.py

```diff
@@ -0,0 +1,54 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Sequence
+
+import requests
+import tarfile
+from io import BytesIO
+
+SCHEMA_URL = "https://github.com/mozilla/mozilla-pipeline-schemas/archive/generated-schemas.tar.gz"
+
+
+@dataclass
+class LegacyPing:
+    name: str
+    versions: Sequence[str] = list
+
+    @property
+    def bigquery_fully_qualified_names(self) -> Sequence[str]:
+        table_names = []
+        for version in self.versions:
+            table_names.append(
+                f"moz-fx-data-shared-prod.telemetry_live."
+                + self.name.replace("-", "_")
+                + "_v"
+                + version.split(".")[1]
+            )
+        return table_names
+
+
+def get_ping_schemas():
+    schemas = requests.get(SCHEMA_URL)
```

Comment:
Could use a docstring too to make it clear what the dict being returned is
```suggestion
def _get_ping_schemas() -> Dict[str, str]:
    schemas = requests.get(SCHEMA_URL)
```